app/models/article.rb

```diff
@@ -342,6 +342,11 @@ def tag_keywords_for_search
     tags.pluck(:keywords_for_search).join
   end
 
+  def long_publish_time
```

Comment:
I would use a past tense here. The normal times are all `crossposted_at`, `published_at`, `updated_at` and so on.

So maybe something like `full_published_at` or `long_published_at` something like that.

I also wonder if this and the other method in `Comment` shouldn't be in the respective decorators: [article_decorator.rb](https://github.com/thepracticaldev/dev.to/blob/master/app/decorators/article_decorator.rb) and [comment_decorator](https://github.com/thepracticaldev/dev.to/blob/master/app/decorators/comment_decorator.rb) - this because it's just presentation logic, not really part of the business logic.

Reply:
I was following the example set by `readable_publish_date`, which lives in both the Comment and Article models, but I agree a decorator makes more sense for both of them. I don't think moving the existing `readable_publish_date` methods should be part of this PR, but I will move my new methods into decorators. There don't seem to be any specs for decorators, should there be?

Reply:
Hmmm I know where @rhymes is going with this but I feel like the `..._at` protocol like `created_at`, `updated_at` should possibly be reserved for values which are a raw timestamp where any time can be calculated from.

Where this might be better the case for a different naming convention. I'd guess that was the original logic at least. However, that isn't to say the current convention which @acflint was following is the correct one.

Yeah, decorator makes sense. We don't have current specs there. We definitely could. Decorator code is typically decently well-covered by request specs, but unit tests couldn't hurt!